chapters/04-estruturas-de-dados.md

```diff
@@ -55,7 +55,7 @@ null.length;
 // → TypeError: Cannot read property 'length' of null
 ```
 
-Os dois modos mais comuns de acessar propriedades em JavaScript são com ponto e colchetes. Ambos _value.index_ e _value[index]_ acessam uma propriedade em _value_, mas não necessariamente a mesma propriedade. A diferença está em como _index_ é interpretada. Ao usar um ponto, a parte após o ponto (que deve ser um nome de variável válido) acessa diretamente o nome da propriedade. Ao usar colchetes, o _índex_ é tratado como uma expressão que é avaliada para obter o nome da propriedade. Considerando que _value.index_ busca a propriedade chamada _"index"_, o _value[index]_ tenta obter o valor da variável chamada _índex_ e então usa isso como o nome da propriedade.
+_Arrays_ também tem uma propriedade length, mantendo a quantidade de elementos no _array_. Na verdade, os elementos no _array_ também são acessados por meio de propriedades. Ambos `value.index` e `value[index]` acessam uma propriedade em value. A diferença está em como _index_ é interpretada. Ao usar um ponto, a parte após o ponto (que deve ser um nome de variável válido) acessa diretamente o nome da propriedade. Ao usar colchetes, o _index_ é tratado como uma expressão que é avaliada para obter o nome da propriedade. Considerando que `value.index` busca a propriedade chamada "index", `value[index]` tenta obter o valor da variável chamada _index_, e usa isso como o nome da propriedade.
```

Comment:
**original:**
> The two most common ways to access properties in JavaScript are with a dot and with square brackets. Both value.x and value[x] access a property on value—but not necessarily the same property. The difference is in how x is interpreted. When using a dot, the part after the dot must be a valid variable name, and it directly names the property. When using square brackets, the expression between the brackets is evaluated to get the property name. Whereas value.x fetches the property of value named “x”, value[x] tries to evaluate the expression x and uses the result as the property name.

**ótimas** melhorias aqui, só a alteração no trecho do início que não está congruente com o original:

> _Arrays_ também tem uma propriedade length, mantendo a quantidade de elementos no _array_. Na verdade, os elementos no _array_ também são acessados por meio de propriedades.

qual o motivo da mudança nesse trecho específico?